prompting/validators/tasks.py

```diff
@@ -155,35 +202,46 @@ def create_qg_task(base_text: str, index: int) -> QuestionGenerationTask:
         ),
     ]
 
-    sampled_criterias = random.sample(possible_criterias, 1)
+    random_sampled_criteria = random.sample(other_random_criteria, 1)
+    criteria = [
+        question_starts_with_prefix_criteria,
+        question_ends_with_criteria,
+    ] + random_sampled_criteria
 
     return QuestionGenerationTask(
         base_text=base_text,
-        criteria=sampled_criterias,
+        criteria=criteria,
         task_type="question-generation",
         task_name=f"followup{index}",
     )
 
 
 def create_qa_task(base_text: str, index: int) -> QuestionAnswerTask:
-    possible_criterias = [
-        MatchLengthCriteria(
-            penalty=0.1,
-            target_length=random.randint(50, 200),
-            unit=TextLengthUnitEnum.WORDS,
-        ),
-        MatchLengthCriteria(
-            penalty=0.1,
-            target_length=random.randint(4, 8),
-            unit=TextLengthUnitEnum.SENTENCES,
-        ),
-    ]
+    answer_should_not_include_criteria = MatchContentCriteria(
```

Comment:
what does n_words mean in this context? The probability of this criteria being applied?

Reply:
The amount of words you want to sample, in order to not bloat the prompt with an extensive list of possibilities. So for the question generation for example, we provide a list of possible `question_prefixes` to the criteria and the `n_words` will random select from this list n samples to be considered in the prompt.

We could drop this parameter and do the sample outside of the criteria as well

Reply:
no i like it. so with a value of 0.2 it will select 20% of the elements of the list?